src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -41,20 +49,15 @@ internal static ImageGrayscalingEstimator ConvertToGrayscale(this TransformsCata
         }
 
         /// <summary>
-        /// Loads the images from the <see cref="ImageLoadingTransformer.ImageFolder" /> into memory.
+        /// Create a <see cref="ImageLoadingEstimator"/>, which loads the data from the column specified in <paramref name="inputColumnName"/>
+        /// as an image to a new column: <paramref name="outputColumnName"/>.
         /// </summary>
-        /// <remarks>
-        /// The image get loaded in memory as a <see cref="System.Drawing.Bitmap" /> type.
-        /// Loading is the first step of almost every pipeline that does image processing, and further analysis on images.
-        /// The images to load need to be in the formats supported by <see cref = "System.Drawing.Bitmap" />.
-        /// For end-to-end image processing pipelines, and scenarios in your applications, see the
-        /// <a href="https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started"> examples in the machinelearning-samples github repository.</a>
-        /// <seealso cref = "ImageEstimatorsCatalog" />
-        /// </remarks>
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="imageFolder">The images folder.</param>
-        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be <see cref="System.Drawing.Bitmap"/>.</param>
+        /// <param name="inputColumnName">Name of the column to copy the data from.
+        /// This estimator operates over text data.</param>
```

Comment:
>over text data [](start = 36, length = 14)

over text data indicating the path of the images to load.

Reply:
I rather put it in sentence above.

---
In reply to: [276905816](https://github.com/dotnet/machinelearning/pull/3376#discussion_r276905816) [](ancestors = 276905816)